inst/python/custom_dp/covariance_custom.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+
+
+def lap(scale):
+    """Compute Laplacian noise"""
+    return np.random.laplace(0, scale)
+
+
+def noisy_average(x, epsilon, x_min, x_max, N, N_noisy):
+    """
+    Implement the NoisyAverage with Normalization algorithm
+    from https://books.google.fr/books?id=WFttDQAAQBAJ&pg=PA29.
+
+    We assume that N and N_noisy are already computed
+    """
+    assert x_min <= x_max, "Verify that you bounds are corrected ordered"
```

Comment:
Even if I'm not sure how reticulate uses Python code, it's usually better to use `if _: raise(_)` than `assert` outside of tests, one reason being that these `asserts` are removed when the compilation is optimized